.github/workflows/binaries.yml

```diff
@@ -0,0 +1,55 @@
+on:
+  release:
+    types: [created]
+name: Binary artifacts
+jobs:
+  generate:
+    env:
+      # The version needs to match with the one used in the printer
+      PRETTIER_API: 2.0.0
+    name: Create release artifacts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          path: 'prettier-java'
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+      - name: Install tools
+        run: |
+          sudo apt update
+          sudo apt install jq
+          npm i -g pkg
+      - name: Installing/Linking dependencies
+        run: |
+          cd $GITHUB_WORKSPACE/prettier-java/packages/java-parser
+          npm i
+          npm link
+          cd $GITHUB_WORKSPACE/prettier-java/packages/prettier-plugin-java
+          npm i
+          npm link java-parser
+      - name: Checkout Prettier
+        uses: actions/checkout@v2
+        with:
+          repository: prettier/prettier
+          ref: refs/tags/${{ env.PRETTIER_API }}
+          path: prettier
+      - name: Link Prettier-java to Prettier
+        run: |
+          cd $GITHUB_WORKSPACE/prettier
+          npm i
+          npm link prettier-plugin-java
+          mv package.json package.json.old
+          jq '.dependencies["prettier-plugin-java"] = "./node_modules/prettier-plugin-java"' package.json.old | cat > package.json
```

Comment:
I do not understand this line: is the `npm link prettier-plugin-java` command not sufficient ? If not, can you add a comment to add some explanation ?

Reply:
It looks like pkg is solely relying on the package.json and does not seems to use node_modules folder. I had to manually route the dependency to the right one.
However this is just my two cents guess as I only did some tests and don't really know how pkg works from the inside.